content/release-notes/2023-02-06-console.md

```diff
@@ -12,7 +12,7 @@
     The previous method of enabling connection pooling for a compute endpoint is deprecated. When switching to the new per-connection method, be sure to disable connection pooling at the compute endpoint. You can do so by [editing the compute endpoint](/docs/manage/endpoints#edit-a-compute-endpoint).
 - UI: The [Neon Free Tier](/docs/introduction/technical-preview-free-tier) now includes a compute endpoint with each branch. Previously, there was a limit of 3 endpoints per project. This limit was removed based on user feedback indicating that it was not conducive to developer workflows that create a branch for each preview deployment.
 
-    Instead of limiting the number of endpoint computes in the Neon Free Tier, there is now a limit of 100 compute hours per month. Currently, the compute hour limit applies to newly created projects only. The limit does not yet apply to existing projects. We expect to apply the limit to all projects later this quarter. Regardless of the compute hour limit, you are always able to connect to the compute endpoint associated with the primary branch of your Neon project. For more information, see [Technical Preview Free Tier](/docs/introduction/technical-preview-free-tier).
+    Instead of limiting the number of endpoint computes in the Neon Free Tier, there is now a limit of 100 compute active time hours per month. Currently, the compute hour limit applies to newly created projects only. The limit does not yet apply to existing projects. We expect to apply the limit to all projects later this quarter. Regardless of the compute hour limit, you are always able to connect to the compute endpoint associated with the primary branch of your Neon project. For more information, see [Technical Preview Free Tier](/docs/introduction/technical-preview-free-tier).
```

Comment:
Hmm, I would expect this to be removed if we have the same info in the 24 Feb note (which is the correct date when this was rolled out to production)

Reply:
Thank you. Meant to remove this one.